apps/ledger-live-desktop/tests/models/SendModal.ts

```diff
@@ -6,12 +6,33 @@ export class SendModal extends Modal {
 
   readonly drowdownAccount: Locator;
   readonly recipientInput: Locator;
+  readonly continueButton: Locator;
+  readonly verifyTotalDebit: Locator;
```

Comment:
It's misleading to call element on the screen "verify" (same goes for "check"), could you please just state element such as instead of "verifyTotalDebit" let's have "totalDebitValue" or "totalDebitText"